core/services/gateway/handler_factory.go

```diff
@@ -33,6 +35,8 @@ func (hf *handlerFactory) NewHandler(handlerType HandlerType, handlerConfig json
 	switch handlerType {
 	case FunctionsHandlerType:
 		return functions.NewFunctionsHandlerFromConfig(handlerConfig, donConfig, don, hf.legacyChains, hf.ds, hf.lggr)
+	case workflowHandlerType:
```

Comment:
@jinhoonbang is already adding it in his PR, maybe wait for that to be merged and then rebase.